src/main/java/iudx/file/server/auditing/AuditingServiceImpl.java

```diff
@@ -1,42 +1,43 @@
 package iudx.file.server.auditing;
 
-import static iudx.file.server.auditing.util.Constants.DATABASE_TABLE_NAME;
-import static iudx.file.server.auditing.util.Constants.ERROR;
-import static iudx.file.server.auditing.util.Constants.FAILED;
-import static iudx.file.server.auditing.util.Constants.MESSAGE;
-import static iudx.file.server.auditing.util.Constants.QUERY_KEY;
-import static iudx.file.server.auditing.util.Constants.SUCCESS;
-
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import io.vertx.core.AsyncResult;
 import io.vertx.core.Future;
 import io.vertx.core.Handler;
-import io.vertx.core.Promise;
 import io.vertx.core.Vertx;
 import io.vertx.core.json.JsonObject;
 import io.vertx.pgclient.PgConnectOptions;
 import io.vertx.pgclient.PgPool;
 import io.vertx.sqlclient.PoolOptions;
 import iudx.file.server.auditing.util.QueryBuilder;
 import iudx.file.server.auditing.util.ResponseBuilder;
+import iudx.file.server.common.Response;
+import iudx.file.server.databroker.DataBrokerService;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+import static iudx.file.server.auditing.util.Constants.EXCHANGE_NAME;
+import static iudx.file.server.auditing.util.Constants.ROUTING_KEY;
+import static iudx.file.server.common.Constants.DATABROKER_SERVICE_ADDRESS;
+
 public class AuditingServiceImpl implements AuditingService {
 
   private static final Logger LOGGER = LogManager.getLogger(AuditingServiceImpl.class);
+  private final QueryBuilder queryBuilder = new QueryBuilder();
+  public static DataBrokerService rmqService;
+  private final ObjectMapper objectMapper = new ObjectMapper();
   PgConnectOptions connectOptions;
   PoolOptions poolOptions;
   PgPool pool;
-  private final QueryBuilder queryBuilder = new QueryBuilder();
-  private JsonObject query = new JsonObject();
+  private JsonObject writeMessage = new JsonObject();
   private String databaseIP;
   private int databasePort;
   private String databaseName;
   private String databaseUserName;
   private String databasePassword;
   private String databaseTableName;
   private int databasePoolSize;
```

Comment:
what is the use of this declaration. not being used

Reply:
> what is the use of this declaration. not being used

Solved.